src/main/java/Todos.java

```diff
@@ -0,0 +1,22 @@
+public class Todos extends Task{
+    Todos(String description) {
+        super(description);
+        System.out.println("\tGot it. I've added this task:");
+        System.out.println("\t\t" + getStatusAndDescription());
+    }
+
+    /**
+     * Do not have to worry about
+     * 1. Lines.
+     * 2. Number of variables in the list.
+     */
+//    public void printAddedTodo() {
+//        System.out.println("\tGot it. I've added this task:");
+//        System.out.println("\t\t" + getStatusAndDescription());
+//    }
```

Comment:
I like how you have given comments for the **@Override getStatusAndDescription** method along with the alternative method to execute the required output. I also had doubt how to @Override methods while implementing my own iP but this would be definitely useful for others who read your code. Well done!